controllers/user.js

```diff
@@ -4,13 +4,16 @@ const Users = require("../models/users");
 
 const router = express.Router();
 
+const validation = require("../middleware/dataValidation");
+const { userSchema } = require("../schemas");
+
 router
   .route("/")
   .get(async (req, res) => {
     const users = await Users.find();
     res.status(200).json({ users });
   })
-  .post(async (req, res) => {
+  .post(validation(userSchema), async (req, res) => {
```

Comment:
This is a perfect example of easy to implement and use once the schemas have been set up -- it should be included for the PUT request below as well, though

Reply:
Somehow mis-targeted my merge on my machine, should be there now.